Sources/SKSwiftPMWorkspace/SwiftPMBuildSystem.swift

```diff
@@ -406,7 +415,7 @@ extension SwiftPMBuildSystem: SKCore.BuildSystem {
       return [ConfiguredTarget(targetID: "", runDestinationID: "dummy")]
     }
 
-    if url.pathExtension == "h", let target = try? target(forHeader: path) {
+    if let target = try? target(forHeader: path) {
```

Comment:
Ah, I noticed the comment in the other PR before this one:
> Did you also rename `target(forHeader:)`? Maybe `inferTarget(for:)`?

Reply:
Renamed to `inferredTarget(for:)`.